cmd/govim/signs.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"fmt"
+)
+
+// Using a sign group creates a separate namespace for all signs placed by govim
+const signGroup = "govim"
+
+type signDef struct {
+	name          string // Name of the sign
+	text          string // One or two chars shown in the gutter
+	textHighlight string // Highlight used
+}
+
+// Only one sign type at the moment, errors.
+var errorSign = signDef{"govimerr", ">>", "Error"}
+
+// signDefine defines the govim error sign and must be called once before placing any signs
+func (v *vimstate) signDefine() error {
+	argDict := struct {
```

Comment:
Why not use `signDef` instead of defining another type here?

Reply:
Leftovers from old implementation, fixed!